deploy/pulse.conf

```diff
@@ -42,6 +33,24 @@ server {
 
   }
 
+  location /data/ {
+      proxy_pass https://s3.amazonaws.com/pulse.cio.gov/live/data/;
+      proxy_http_version 1.1;
+      proxy_redirect off;
+
+      # don't set Host header, Amazon doesn't like this
+      # proxy_set_header Host   $host;
```

Comment:
Amazon S3 returns a 403 forbidden error if you change the `Host` header to something besides `s3.amazonaws.com`, it turns out.